<?xml version="1.0" encoding="UTF-8"?>
<!-- 
//
// Copyright (c) 2021 The University of Aizu
// Open source under the BSD License.
//
 -->
<ui version="4.0">
 <class>LoadDataList</class>
 <widget class="QDialog" name="LoadDataList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="loadedDataList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>339</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="loadFileButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Load file</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="changeParameterButton">
   <property name="geometry">
    <rect>
     <x>202</x>
     <y>430</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply to all window</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>MIN</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="minT"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>to MAX</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="maxT"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Change parameter</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>320</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="tilingButton">
      <property name="text">
       <string>Tiling</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clearListButton">
      <property name="text">
       <string>Clear list</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="closeAllButton">
      <property name="text">
       <string>Close all window</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="changeColor">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Rainbow</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Gray scale</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Iron</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="substract">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Subtract</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>loadFileButton</tabstop>
  <tabstop>loadedDataList</tabstop>
  <tabstop>tilingButton</tabstop>
  <tabstop>clearListButton</tabstop>
  <tabstop>closeAllButton</tabstop>
  <tabstop>minT</tabstop>
  <tabstop>maxT</tabstop>
  <tabstop>changeColor</tabstop>
  <tabstop>changeParameterButton</tabstop>
  <tabstop>substract</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
